corehq/ex-submodules/phonelog/tasks.py

```diff
@@ -27,7 +27,10 @@ def purge_old_device_report_entries():
 
 
 @no_result_task(queue='sumologic_logs_queue', default_retry_delay=10 * 60, max_retries=3, bind=True)
-def send_device_log_to_sumologic(self, url, data, headers):
+def send_device_log_to_sumologic(self, url, data, headers, domain):
```

Comment:
Does this cause a domain tag to be added to DD metrics? Is it called frequently enough that it could have a significant impact on DD tag costs, and do we care about that?

Reply:
I think we are already shipping this tag, it's just going out as N/A at the moment. Are the costs based on how many different tags we pass or the values of the tags?

Reply:
The cost is based on tag values. More [here](https://docs.datadoghq.com/account_management/billing/custom_metrics/).